### PR TITLE
doc: add dataflow schema

### PR DESCRIPTION
**Pull request by flomonster**

### CODE REVIEW

**Comment by Khoyo** [comment]
That's a dataflow schema ;) Thanks @multun for catching that.

File: content/docs/reference/architecture/workflow/workflow.en.svg
Comment (axrolld) [inline content/docs/reference/architecture/workflow/workflow.en.svg:1]: Would it make sense to represent this as a [Data Flow Diagram](https://en.wikipedia.org/wiki/Data-flow_diagram) ? Or an UML equivalent ? I don't know much about those diagrams but i think there's one type that would be a good standard to represent this.

File: content/docs/reference/architecture/workflow/workflow.fr.svg
Comment (axrolld) [inline content/docs/reference/architecture/workflow/workflow.fr.svg:1]: Isn't supposed to be an arrow between the Infra and the editor ? As the editor is able to modify the infra ?
Also, do we want to explain more things about Input STDCM ?

**Review by axrolld — APPROVED** [review]

File: content/docs/reference/architecture/workflow/workflow.fr.svg
Comment (flomonster) [reply content/docs/reference/architecture/workflow/workflow.fr.svg:1]: Fixed.

**Review by axrolld — APPROVED** [review]

**Pull request merged**